examples/js/loaders/MTLLoader.js

```diff
@@ -390,6 +441,18 @@ THREE.MTLLoader.MaterialCreator.prototype = {
 
 	},
 
+	_resolveURL: function ( url ) {
```

Comment:
Almost there!

We have also been trying to move away from these kind of public methods whenever possible (people would start using them and get mad when they get removed). Considering that this method is mainly used inside ` THREE.MTLLoader.MaterialCreator` we could just define it locally.